### ISSUE TITLE
Wink2 - UUID is not valid 

### ISSUE THREAD

**Issue by sb13054** [home-assistant/homebridge-homeassistant#170]
I'm currently running home bridge on pi3.  I get an error when running home bridge.  Included below is my config.json and error log.  I've also attached the AccessoriesCached list.

JSON
```json{
    "bridge": {
            "name": "Homebridge",
            "username": “ab:22:3D:E3:CE:30",
            "port": 51826,
            "pin": "031-45-154"
    },
"platforms": [
   	{
   		"platform": "Wink2",
   		"name": "Wink2",
   		"username": “xyz@gmail.com",
   		"password": “xyzzy”,
   		"client_id": “xxxxxxx”,
   		"client_secret": “xxxxxxx”,
   		"hide_groups": ["garage_door", "thermostat"],
   		"hide_ids": [],
   		"fan_ids": []
   	}
   ]
}
```

LOG
```
*** WARNING *** The program 'node' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=node>
*** WARNING *** The program 'node' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=node&f=DNSServiceRegister>
[6/11/2017, 12:59:57 PM] Loaded plugin: homebridge-liftmaster2
[6/11/2017, 12:59:57 PM] Registering platform 'homebridge-liftmaster2.LiftMaster2'
[6/11/2017, 12:59:57 PM] ---
[6/11/2017, 12:59:58 PM] Loaded plugin: homebridge-nest
[6/11/2017, 12:59:58 PM] Registering platform 'homebridge-nest.Nest'
[6/11/2017, 12:59:58 PM] ---
[6/11/2017, 1:00:00 PM] Loaded plugin: homebridge-wink2
[6/11/2017, 1:00:00 PM] Registering platform 'homebridge-wink2.Wink2'
[6/11/2017, 1:00:00 PM] ---
[6/11/2017, 1:00:00 PM] Loaded config.json with 0 accessories and 1 platforms.
[6/11/2017, 1:00:00 PM] ---
[6/11/2017, 1:00:00 PM] Loading 1 platforms...
[6/11/2017, 1:00:00 PM] [Wink2] Initializing Wink2 platform...
[6/11/2017, 1:00:00 PM] [Wink2] Wink2 Init - Version 2.0.8
Load homebridge-liftmaster2.LiftMaster2
Scan this code with your HomeKit App on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 031-45-154 │     
    └────────────┘     
                       
[6/11/2017, 1:00:00 PM] [Wink2] Reclaiming: Akshar's Light - ID 297278 - TYPE binary_switch
[6/11/2017, 1:00:00 PM] [Wink2] Reclaiming: Akshar's Fan - ID 297279 - TYPE binary_switch
[6/11/2017, 1:00:00 PM] [Wink2] Reclaiming: Media Room - ID 299589 - TYPE binary_switch
[6/11/2017, 1:00:00 PM] [Wink2] Reclaiming: Media Room - ID 299590 - TYPE binary_switch
[6/11/2017, 1:00:00 PM] [Wink2] Reclaiming: Bottom Light Load - ID 470043 - TYPE binary_switch
[6/11/2017, 1:00:00 PM] [Wink2] Reclaiming: Top Light Load - ID 470044 - TYPE binary_switch
[6/11/2017, 1:00:00 PM] [Wink2] Direct Access NOT Enabled. Add direct_access:true to your config if you which to use direct access on Wink.
[6/11/2017, 1:00:00 PM] [homebridge-liftmaster2.LiftMaster2] Please setup MyQ login information!
[6/11/2017, 1:00:00 PM] Homebridge is running on port 51826.
[6/11/2017, 1:00:01 PM] [Wink2] Wink Authentication Successful.
/usr/local/lib/node_modules/homebridge/lib/platformAccessory.js:17
  if (!uuid.isValid(UUID)) throw new Error("UUID '" + UUID + "' is not a valid UUID. Try using the provided 'generateUUID' function to create a valid UUID from any arbitrary string, like a serial number.");
                           ^

Error: UUID '924f20c0-eb15-0131-e510-1e5449afed4e' is not a valid UUID. Try using the provided 'generateUUID' function to create a valid UUID from any arbitrary string, like a serial number.
    at new PlatformAccessory (/usr/local/lib/node_modules/homebridge/lib/platformAccessory.js:17:34)
    at Wink2.addAccessoryCharacteristics (/usr/local/lib/node_modules/homebridge-wink2/index.js:133:24)
    at Wink2.addAccessory (/usr/local/lib/node_modules/homebridge-wink2/index.js:123:10)
    at winkapiv2._deviceReceiptProcessor (/usr/local/lib/node_modules/homebridge-wink2/lib/winkapi.js:222:22)
    at winkapiv2.<anonymous> (/usr/local/lib/node_modules/homebridge-wink2/lib/winkapi.js:119:28)
    at ClientRequest.<anonymous> (/usr/local/lib/node_modules/homebridge-wink2/lib/winkapi.js:361:17)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:186:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

ACCESSORIES CACHED
```json
[
  {
    "plugin": "homebridge-Wink2",
    "platform": "Wink2",
    "displayName": "Akshar's Light",
    "UUID": "c9cc7f42-1638-465f-9d99-f7a2642f2eab",
    "category": 1,
    "context": {
      "object_type": "binary_switch",
      "object_id": "297278",
      "uuid": "c9cc7f42-1638-465f-9d99-f7a2642f2eab",
      "icon_id": "52",
      "icon_code": "binary_switch-light_bulb_dumb",
      "desired_state": {
        "powered": false,
        "powering_mode": "dumb"
      },
      "last_reading": {
        "connection": true,
        "connection_updated_at": 1497150917.7448835,
        "powered": false,
        "powered_updated_at": 1497150917.7448835,
        "powering_mode": "dumb",
        "powering_mode_updated_at": 1478901145.1660974,
        "desired_powered_updated_at": 1493858958.4050698,
        "desired_powering_mode_updated_at": 1478901162.973793,
        "powered_changed_at": 1497150917.7448835
      },
      "subscription": {
        "pubnub": {
          "subscribe_key": "sub-c-f7bf7f7e-0542-11e3-a5e8-02ee2ddab7fe",
          "channel": "7b2e3359f8d652454cf5d175745e8ca204095eb6|binary_switch-297278|user-47505"
        }
      },
      "binary_switch_id": "297278",
      "name": "Akshar's Light",
      "locale": "en_us",
      "units": {},
      "created_at": 1478901123,
      "hidden_at": null,
      "capabilities": {
        "fields": [
          {
            "type": "boolean",
            "field": "connection",
            "mutability": "read-only"
          },
          {
            "type": "boolean",
            "field": "powered",
            "mutability": "read-write"
          },
          {
            "type": "selection",
            "field": "powering_mode",
            "choices": [
              "smart",
              "dumb",
              "none"
            ],
            "mutability": "read-write"
          }
        ]
      },
      "triggers": [],
      "manufacturer_device_model": "wink_relay_switch",
      "manufacturer_device_id": null,
      "device_manufacturer": "wink",
      "model_name": "Wink Relay Switch",
      "upc_id": "315",
      "upc_code": "wink_p1_binary_switch",
      "primary_upc_code": "wink_p1_binary_switch",
      "gang_id": "51733",
      "hub_id": "522547",
      "local_id": "1",
      "radio_type": "project_one",
      "linked_service_id": null,
      "current_budget": null,
      "lat_lng": [
        null,
        null
      ],
      "location": "",
      "order": 0
    },
    "services": [
      {
        "UUID": "0000003E-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "Identify",
            "UUID": "00000014-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pw"
              ]
            },
            "value": false
          },
          {
            "displayName": "Manufacturer",
            "UUID": "00000020-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink"
          },
          {
            "displayName": "Model",
            "UUID": "00000021-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink_relay_switch"
          },
          {
            "displayName": "Name",
            "UUID": "00000023-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "Akshar's Light"
          },
          {
            "displayName": "Serial Number",
            "UUID": "00000030-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "297278"
          }
        ]
      },
      {
        "UUID": "00000043-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "On",
            "UUID": "00000025-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr",
                "pw",
                "ev"
              ]
            },
            "value": false
          }
        ]
      }
    ]
  },
  {
    "plugin": "homebridge-Wink2",
    "platform": "Wink2",
    "displayName": "Akshar's Fan",
    "UUID": "ad9d19fc-0b85-42f0-85bf-8712b549ce5d",
    "category": 1,
    "context": {
      "object_type": "binary_switch",
      "object_id": "297279",
      "uuid": "ad9d19fc-0b85-42f0-85bf-8712b549ce5d",
      "icon_id": "19",
      "icon_code": "binary_switch-fan_light",
      "desired_state": {
        "powered": false,
        "powering_mode": "dumb"
      },
      "last_reading": {
        "connection": true,
        "connection_updated_at": 1497053641.4555867,
        "powered": false,
        "powered_updated_at": 1497053641.4555867,
        "powering_mode": "dumb",
        "powering_mode_updated_at": 1478902003.789974,
        "desired_powered_updated_at": 1496919637.071742,
        "desired_powering_mode_updated_at": 1496919780.5811296,
        "powered_changed_at": 1496919636.8288972,
        "desired_powered_changed_at": 1496919637.071742
      },
      "subscription": {
        "pubnub": {
          "subscribe_key": "sub-c-f7bf7f7e-0542-11e3-a5e8-02ee2ddab7fe",
          "channel": "1be0c08700a661907dcbdc441e0987288b26277f|binary_switch-297279|user-47505"
        }
      },
      "binary_switch_id": "297279",
      "name": "Akshar's Fan",
      "locale": "en_us",
      "units": {},
      "created_at": 1478901123,
      "hidden_at": null,
      "capabilities": {
        "fields": [
          {
            "type": "boolean",
            "field": "connection",
            "mutability": "read-only"
          },
          {
            "type": "boolean",
            "field": "powered",
            "mutability": "read-write"
          },
          {
            "type": "selection",
            "field": "powering_mode",
            "choices": [
              "smart",
              "dumb",
              "none"
            ],
            "mutability": "read-write"
          }
        ]
      },
      "triggers": [],
      "manufacturer_device_model": "wink_relay_switch",
      "manufacturer_device_id": null,
      "device_manufacturer": "wink",
      "model_name": "Wink Relay Switch",
      "upc_id": "315",
      "upc_code": "wink_p1_binary_switch",
      "primary_upc_code": "wink_p1_binary_switch",
      "gang_id": "51733",
      "hub_id": "522547",
      "local_id": "2",
      "radio_type": "project_one",
      "linked_service_id": null,
      "current_budget": null,
      "lat_lng": [
        null,
        null
      ],
      "location": "",
      "order": 0
    },
    "services": [
      {
        "UUID": "0000003E-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "Identify",
            "UUID": "00000014-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pw"
              ]
            },
            "value": false
          },
          {
            "displayName": "Manufacturer",
            "UUID": "00000020-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink"
          },
          {
            "displayName": "Model",
            "UUID": "00000021-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink_relay_switch"
          },
          {
            "displayName": "Name",
            "UUID": "00000023-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "Akshar's Fan"
          },
          {
            "displayName": "Serial Number",
            "UUID": "00000030-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "297279"
          }
        ]
      },
      {
        "UUID": "00000043-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "On",
            "UUID": "00000025-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr",
                "pw",
                "ev"
              ]
            },
            "value": false
          }
        ]
      }
    ]
  },
  {
    "plugin": "homebridge-Wink2",
    "platform": "Wink2",
    "displayName": "Media Room",
    "UUID": "82ad47c9-4947-4374-a976-55e86056512a",
    "category": 1,
    "context": {
      "object_type": "binary_switch",
      "object_id": "299589",
      "uuid": "82ad47c9-4947-4374-a976-55e86056512a",
      "icon_id": "52",
      "icon_code": "binary_switch-light_bulb_dumb",
      "desired_state": {
        "powered": false,
        "powering_mode": "none"
      },
      "last_reading": {
        "connection": true,
        "connection_updated_at": 1497059883.7389333,
        "powered": false,
        "powered_updated_at": 1497059883.7389333,
        "powering_mode": "none",
        "powering_mode_updated_at": 1479475464.7401218,
        "desired_powered_updated_at": 1489023022.0265338,
        "desired_powering_mode_updated_at": 1489023021.9985023
      },
      "subscription": {
        "pubnub": {
          "subscribe_key": "sub-c-f7bf7f7e-0542-11e3-a5e8-02ee2ddab7fe",
          "channel": "3cf6bdbeaa63922aff0d81a093879ef8edd41aee|binary_switch-299589|user-47505"
        }
      },
      "binary_switch_id": "299589",
      "name": "Media Room",
      "locale": "en_us",
      "units": {},
      "created_at": 1479083920,
      "hidden_at": null,
      "capabilities": {
        "fields": [
          {
            "type": "boolean",
            "field": "connection",
            "mutability": "read-only"
          },
          {
            "type": "boolean",
            "field": "powered",
            "mutability": "read-write"
          },
          {
            "type": "selection",
            "field": "powering_mode",
            "choices": [
              "smart",
              "dumb",
              "none"
            ],
            "mutability": "read-write"
          }
        ]
      },
      "triggers": [],
      "manufacturer_device_model": "wink_relay_switch",
      "manufacturer_device_id": null,
      "device_manufacturer": "wink",
      "model_name": "Wink Relay Switch",
      "upc_id": "315",
      "upc_code": "wink_p1_binary_switch",
      "primary_upc_code": "wink_p1_binary_switch",
      "gang_id": "52114",
      "hub_id": "524514",
      "local_id": "1",
      "radio_type": "project_one",
      "linked_service_id": null,
      "current_budget": null,
      "lat_lng": [
        null,
        null
      ],
      "location": "",
      "order": 0
    },
    "services": [
      {
        "UUID": "0000003E-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "Identify",
            "UUID": "00000014-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pw"
              ]
            },
            "value": false
          },
          {
            "displayName": "Manufacturer",
            "UUID": "00000020-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink"
          },
          {
            "displayName": "Model",
            "UUID": "00000021-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink_relay_switch"
          },
          {
            "displayName": "Name",
            "UUID": "00000023-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "Media Room"
          },
          {
            "displayName": "Serial Number",
            "UUID": "00000030-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "299589"
          }
        ]
      },
      {
        "UUID": "00000043-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "On",
            "UUID": "00000025-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr",
                "pw",
                "ev"
              ]
            },
            "value": false
          }
        ]
      }
    ]
  },
  {
    "plugin": "homebridge-Wink2",
    "platform": "Wink2",
    "displayName": "Media Room",
    "UUID": "931643eb-54c8-4386-9ea8-c30db20ff2f5",
    "category": 1,
    "context": {
      "object_type": "binary_switch",
      "object_id": "299590",
      "uuid": "931643eb-54c8-4386-9ea8-c30db20ff2f5",
      "icon_id": "52",
      "icon_code": "binary_switch-light_bulb_dumb",
      "desired_state": {
        "powered": false,
        "powering_mode": "dumb"
      },
      "last_reading": {
        "connection": true,
        "connection_updated_at": 1497189303.6178863,
        "powered": false,
        "powered_updated_at": 1497189303.6178863,
        "powering_mode": "dumb",
        "powering_mode_updated_at": 1479475456.453833,
        "desired_powered_updated_at": 1497189431.9108315,
        "desired_powering_mode_updated_at": 1497189431.9108315,
        "powered_changed_at": 1497189303.6178863,
        "desired_powered_changed_at": 1497189431.9108315
      },
      "subscription": {
        "pubnub": {
          "subscribe_key": "sub-c-f7bf7f7e-0542-11e3-a5e8-02ee2ddab7fe",
          "channel": "a2a9f1006624fd5e794e0c862f3d6f9ec9aa9b7e|binary_switch-299590|user-47505"
        }
      },
      "binary_switch_id": "299590",
      "name": "Media Room",
      "locale": "en_us",
      "units": {},
      "created_at": 1479083920,
      "hidden_at": null,
      "capabilities": {
        "fields": [
          {
            "type": "boolean",
            "field": "connection",
            "mutability": "read-only"
          },
          {
            "type": "boolean",
            "field": "powered",
            "mutability": "read-write"
          },
          {
            "type": "selection",
            "field": "powering_mode",
            "choices": [
              "smart",
              "dumb",
              "none"
            ],
            "mutability": "read-write"
          }
        ]
      },
      "triggers": [],
      "manufacturer_device_model": "wink_relay_switch",
      "manufacturer_device_id": null,
      "device_manufacturer": "wink",
      "model_name": "Wink Relay Switch",
      "upc_id": "315",
      "upc_code": "wink_p1_binary_switch",
      "primary_upc_code": "wink_p1_binary_switch",
      "gang_id": "52114",
      "hub_id": "524514",
      "local_id": "2",
      "radio_type": "project_one",
      "linked_service_id": null,
      "current_budget": null,
      "lat_lng": [
        null,
        null
      ],
      "location": "",
      "order": 0
    },
    "services": [
      {
        "UUID": "0000003E-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "Identify",
            "UUID": "00000014-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pw"
              ]
            },
            "value": false
          },
          {
            "displayName": "Manufacturer",
            "UUID": "00000020-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink"
          },
          {
            "displayName": "Model",
            "UUID": "00000021-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink_relay_switch"
          },
          {
            "displayName": "Name",
            "UUID": "00000023-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "Media Room"
          },
          {
            "displayName": "Serial Number",
            "UUID": "00000030-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "299590"
          }
        ]
      },
      {
        "UUID": "00000043-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "On",
            "UUID": "00000025-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr",
                "pw",
                "ev"
              ]
            },
            "value": false
          }
        ]
      }
    ]
  },
  {
    "plugin": "homebridge-Wink2",
    "platform": "Wink2",
    "displayName": "Bottom Light Load",
    "UUID": "85e4d426-f808-4864-bdd6-f751af2429a1",
    "category": 1,
    "context": {
      "object_type": "binary_switch",
      "object_id": "470043",
      "uuid": "85e4d426-f808-4864-bdd6-f751af2429a1",
      "icon_id": "52",
      "icon_code": "binary_switch-light_bulb_dumb",
      "desired_state": {
        "powered": false,
        "powering_mode": "none"
      },
      "last_reading": {
        "connection": true,
        "connection_updated_at": 1497096070.8111963,
        "powered": false,
        "powered_updated_at": 1497096070.8111963,
        "powering_mode": "none",
        "powering_mode_updated_at": 1495072751.558069,
        "desired_powered_updated_at": 1495072769.935519,
        "desired_powering_mode_updated_at": 1495072751.7715752,
        "desired_powered_changed_at": null,
        "desired_powering_mode_changed_at": null
      },
      "subscription": {
        "pubnub": {
          "subscribe_key": "sub-c-f7bf7f7e-0542-11e3-a5e8-02ee2ddab7fe",
          "channel": "45fea8c714c2899dc514beb014f4f13f987c4b4b|binary_switch-470043|user-47505"
        }
      },
      "binary_switch_id": "470043",
      "name": "Bottom Light Load",
      "locale": "en_us",
      "units": {},
      "created_at": 1495072321,
      "hidden_at": null,
      "capabilities": {
        "fields": [
          {
            "type": "boolean",
            "field": "connection",
            "mutability": "read-only"
          },
          {
            "type": "boolean",
            "field": "powered",
            "mutability": "read-write"
          },
          {
            "type": "selection",
            "field": "powering_mode",
            "choices": [
              "smart",
              "dumb",
              "none"
            ],
            "mutability": "read-write"
          }
        ]
      },
      "triggers": [],
      "manufacturer_device_model": "wink_relay_switch",
      "manufacturer_device_id": null,
      "device_manufacturer": "wink",
      "model_name": "Wink Relay Switch",
      "upc_id": "315",
      "upc_code": "wink_p1_binary_switch",
      "primary_upc_code": "wink_p1_binary_switch",
      "gang_id": "81801",
      "hub_id": "651988",
      "local_id": "2",
      "radio_type": "project_one",
      "linked_service_id": null,
      "current_budget": null,
      "lat_lng": [
        null,
        null
      ],
      "location": "",
      "order": 0
    },
    "services": [
      {
        "UUID": "0000003E-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "Identify",
            "UUID": "00000014-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pw"
              ]
            },
            "value": false
          },
          {
            "displayName": "Manufacturer",
            "UUID": "00000020-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink"
          },
          {
            "displayName": "Model",
            "UUID": "00000021-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink_relay_switch"
          },
          {
            "displayName": "Name",
            "UUID": "00000023-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "Bottom Light Load"
          },
          {
            "displayName": "Serial Number",
            "UUID": "00000030-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "470043"
          }
        ]
      },
      {
        "UUID": "00000043-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "On",
            "UUID": "00000025-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr",
                "pw",
                "ev"
              ]
            },
            "value": false
          }
        ]
      }
    ]
  },
  {
    "plugin": "homebridge-Wink2",
    "platform": "Wink2",
    "displayName": "Top Light Load",
    "UUID": "b72c9a02-75cf-4e04-86db-8f2a5dbc1999",
    "category": 1,
    "context": {
      "object_type": "binary_switch",
      "object_id": "470044",
      "uuid": "b72c9a02-75cf-4e04-86db-8f2a5dbc1999",
      "icon_id": "52",
      "icon_code": "binary_switch-light_bulb_dumb",
      "desired_state": {
        "powered": false,
        "powering_mode": "none"
      },
      "last_reading": {
        "connection": true,
        "connection_updated_at": 1497096070.7493742,
        "powered": false,
        "powered_updated_at": 1497096070.7493742,
        "powering_mode": "none",
        "powering_mode_updated_at": 1495072745.693341,
        "desired_powered_updated_at": 1495072769.9791594,
        "desired_powering_mode_updated_at": 1495072745.8012493,
        "desired_powered_changed_at": null,
        "desired_powering_mode_changed_at": null
      },
      "subscription": {
        "pubnub": {
          "subscribe_key": "sub-c-f7bf7f7e-0542-11e3-a5e8-02ee2ddab7fe",
          "channel": "binary_switch-470044|f55bb63793adb3a549c210a963851268873efa9d|user-47505"
        }
      },
      "binary_switch_id": "470044",
      "name": "Top Light Load",
      "locale": "en_us",
      "units": {},
      "created_at": 1495072321,
      "hidden_at": null,
      "capabilities": {
        "fields": [
          {
            "type": "boolean",
            "field": "connection",
            "mutability": "read-only"
          },
          {
            "type": "boolean",
            "field": "powered",
            "mutability": "read-write"
          },
          {
            "type": "selection",
            "field": "powering_mode",
            "choices": [
              "smart",
              "dumb",
              "none"
            ],
            "mutability": "read-write"
          }
        ]
      },
      "triggers": [],
      "manufacturer_device_model": "wink_relay_switch",
      "manufacturer_device_id": null,
      "device_manufacturer": "wink",
      "model_name": "Wink Relay Switch",
      "upc_id": "315",
      "upc_code": "wink_p1_binary_switch",
      "primary_upc_code": "wink_p1_binary_switch",
      "gang_id": "81801",
      "hub_id": "651988",
      "local_id": "1",
      "radio_type": "project_one",
      "linked_service_id": null,
      "current_budget": null,
      "lat_lng": [
        null,
        null
      ],
      "location": "",
      "order": 0
    },
    "services": [
      {
        "UUID": "0000003E-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "Identify",
            "UUID": "00000014-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pw"
              ]
            },
            "value": false
          },
          {
            "displayName": "Manufacturer",
            "UUID": "00000020-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink"
          },
          {
            "displayName": "Model",
            "UUID": "00000021-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "wink_relay_switch"
          },
          {
            "displayName": "Name",
            "UUID": "00000023-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "Top Light Load"
          },
          {
            "displayName": "Serial Number",
            "UUID": "00000030-0000-1000-8000-0026BB765291",
            "props": {
              "format": "string",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr"
              ]
            },
            "value": "470044"
          }
        ]
      },
      {
        "UUID": "00000043-0000-1000-8000-0026BB765291",
        "characteristics": [
          {
            "displayName": "On",
            "UUID": "00000025-0000-1000-8000-0026BB765291",
            "props": {
              "format": "bool",
              "unit": null,
              "minValue": null,
              "maxValue": null,
              "minStep": null,
              "perms": [
                "pr",
                "pw",
                "ev"
              ]
            },
            "value": false
          }
        ]
      }
    ]
  }
]
```

**Comment by robbiet480** [comment]
This could be fixed via #165.

**Comment by schmittx** [comment]
Closing this issue due to inactivity, if you're still having problems please open a new issue.
